.circleci/config.yml

```diff
@@ -704,6 +704,51 @@ step-save-src-cache: &step-save-src-cache
     key: v5-src-cache-{{ arch }}-{{ checksum "src/electron/.depshash" }}
     name: Persisting src cache
 
+# Check for doc only change
+step-check-for-doc-only-change: &step-check-for-doc-only-change
+  run:
+    name: Check if commit is doc only change
+    command: |
+      cd src/electron
+      node script/yarn install --frozen-lockfile
+      if node script/doc-only-change.js --prNumber=$CIRCLE_PR_NUMBER --prURL=$CIRCLE_PULL_REQUEST; then
+        #PR is not a doc only change; create empty file to indicate check has been done
+        touch .skip-ci-build
```

Comment:
my first thought seeing this file name is that its _presence_ indicates we should skip the build, rather than its content. I don't see anything that uses the presence of an empty file as an indicator, so can we use the `-e` rather than `-s`?

Reply:
Yeah, my first thought was to look for file existence, but `step-persist-doc-only-change` fails if the file doesn't exist 😦

Reply:
😭